default.py

```diff
@@ -220,13 +220,37 @@ def PLANNED(payload, params):
 def PLANNED_PAGES(payload, params):
     return control.draw_items(_BROWSER.get_watchlist_planned(int(payload)), unbookmark_cm)
 
+@route('search_history')
+def SEARCH(payload, params):
+    history_delimiter=":_:"
+    history = control.getSetting("9anime.history")
+    history_array = history.split(history_delimiter)
+    if history != "" and "Yes" in control.getSetting('searchhistory') :
+        return control.draw_items(_BROWSER.search_history(history_array))
+    else :
+        return SEARCH(payload,params)
+
 @route('search')
 def SEARCH(payload, params):
+    history_delimiter=":_:"
```

Comment:
can we define this globally?
HISTORY_DELIM

Reply:
Sure thing